datumaro/plugins/yolo_format/extractor.py

```diff
@@ -93,31 +107,29 @@ def __init__(
         #
         # To support more subset names, we disallow subsets
         # called 'classes', 'names' and 'backup'.
-        subsets = {k: v for k, v in config.items() if k not in YoloPath.RESERVED_CONFIG_KEYS}
+        subsets = {k: v for k, v in self._config.items() if k not in self.RESERVED_CONFIG_KEYS}
 
         for subset_name, list_path in subsets.items():
```

Comment:
If we know the total number of item to load, please add use of the `self._ctx.progress_reporter` to report import progress here.

Reply:
Added it

Reply:
And now I thought about it a bit, and it seems there already exists progress reporter for import, here: https://github.com/cvat-ai/datumaro/pull/50/files#diff-e2353d651bb7fa627889c3bab9c9e2d6002415a5fb5d70fbec7f1d4274908995R308

Reply:
Ok, sorry for the extra work then.

Reply:
rolled it back